hbp_knowledge/receptors/zoli2018.bel

```diff
@@ -83,214 +92,337 @@ a(MESH:Acetylcholine) -- bp(MESH:Apoptosis)
+SET Confidence = "Medium"
 SET MeSHAnatomy ="GABAergic Neurons"
 a(MESH:Nicotine) -> a(MESH:Glutamates)
 a(MESH:Glutamates) -> bp(MESH:"Long-Term Potentiation")
 a(MESH:Glutamates) -> bp(GO:"action potential initiation")
+UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence =" By activating the α4β2 receptors on inhibitory GABAergic inputs to the VTA or GABAergic interneurons, smoked concentrations of nicotine transiently increase the release of GABA and subse- quently depress it for about one hour "
+SET Confidence = "Low"
 a(MESH:Nicotine) -> complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2))
 a(MESH:Nicotine) -> p(MESH:"Receptors, GABA")
 a(MESH:Nicotine) -| p(MESH:"Receptors, GABA")
+#without temporal factor, above two statements are conflicting
+UNSET Confidence
 
 SET Evidence ="nicotine modulates the shift towards burst firing and increases DA release in the NAc "
+SET Confidence = "Medium"
 a(MESH:Nicotine) -> bp(GO:"action potential initiation")
-a(MESH:Nicotine) -> a(MESH:Dopamine)
-
+a(MESH:Nicotine) -> a(MESH:Dopamine,loc(MESH:"Nucleus Accumbens"))
+UNSET Confidence
 
 SET Evidence ="Chronic nicotine treatment also activates the α7 receptors expressed on glutamatergic terminals, increases the release of glutamate (which facilitates the burst firing of VTA DA neurons), increases NMDA receptor activity, and LTP [79], but simultaneosusly induces the desensitisation of the α4β2 receptors on GABAergic terminals. Overall, these effects decrease the inhibition onto DA neurons, and increase DA release in the NAc [82]."
+SET Confidence = "Medium"
 a(MESH:Nicotine) -> act(p(HGNC:CHRNA7))
 a(MESH:Nicotine) -> a(MESH:Glutamates)
-a(MESH:Nicotine) -> p(MESH:"Receptors, N-Methyl-D-Aspartate")
+a(MESH:Nicotine) -> act(p(MESH:"Receptors, N-Methyl-D-Aspartate"))
 a(MESH:Nicotine) -> bp(MESH:"Long-Term Potentiation")
 a(MESH:Nicotine) -| act(complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)))
-a(MESH:Nicotine) -> a(MESH:Dopamine)
+a(MESH:Nicotine) -> a(MESH:Dopamine,loc(MESH:"Nucleus Accumbens"))
+UNSET Confidence
 
 SET Evidence ="The Hb-IPN system expresses the highest levels and va- riety of nAChR subunits and subtypes in mammalian brain [85], and is the only central system expressing high levels of α3, β4 and α5 subunits. "
+SET Confidence = "Low"
 bp(HBP:"habenulo-interpeduncular pathway") -> p(MESH:"Receptors, Nicotinic")
 bp(HBP:"habenulo-interpeduncular pathway") -> p(HGNC:CHRNA3,loc(MESH:"Central Nervous System"))
 bp(HBP:"habenulo-interpeduncular pathway") -> p(HGNC:CHRNB4,loc(MESH:"Central Nervous System"))
 bp(HBP:"habenulo-interpeduncular pathway") -> p(HGNC:CHRNA5,loc(MESH:"Central Nervous System"))
+#does the evidence suggest the location as central nervous system?
```

Comment:
no, in my opinion it's a pathway that is referred to as central system in the body. So I would delete the location

Reply:
Changes made